lib/widgets/transaction_item.dart

```diff
@@ -13,6 +15,26 @@ class TransactionItem extends StatelessWidget {
   final Transaction transaction;
   final Function deleteTx;
 
+  @override
+  _TransactionItemState createState() => _TransactionItemState();
+}
+
+class _TransactionItemState extends State<TransactionItem> {
+  Color _bgColor;
+
+  @override
+  void initState() {
+    const availableColors = [
+      Colors.purple,
+      Colors.pink,
+      Colors.blue,
+      Colors.green,
+    ];
+
+    _bgColor = availableColors[Random().nextInt(4)];
```

Comment:
can you try `Random().nextInt(availableColors.size)`

Reply:
It throws an error because size isn't defined in the list. 
![image](https://user-images.githubusercontent.com/74462760/101245895-edb37200-374a-11eb-917b-e5411bfad503.png)